openapi/kas-fleet-manager-private.yaml

```diff
@@ -404,41 +363,7 @@ components:
                 type: string
               status:
                 type: string
-        total:
```

Comment:
@pawelpaszki this is where the removal happens as mentioned in the https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/pull/1098/files#r898332632